libs/zedUpload/datastore_http.go

```diff
@@ -79,6 +81,25 @@ func (ep *HttpTransportMethod) WithSrcIPAndProxySelection(localAddr net.IP,
 	return nil
 }
 
+// WithSrcIPAndHTTPSCerts skip verify or append certs for https datastore
+func (ep *HttpTransportMethod) WithSrcIPAndHTTPSCerts(localAddr net.IP, skip bool, certs [][]byte) error {
+	ep.hClient = httpClientSrcIP(localAddr, nil)
```

Comment:
I would create a new function httpClientFoo with this as the body so it can be called for the other transports. 
Note that we currently don't restrict the setting of the added certs to https in the UI AFAIK.

Reply:
sure, i'll rename this.